### DISCUSSION TITLE
Agents

### DISCUSSION THREAD

**Discussion by anwarrassool** [lofcz/LLMTornado#24] [Q&A] [answered]
Does the package support the use of agents with OpenAI?

**Comment by lofcz** [comment, marked as answer]
The support is only partial, I wouldn't recommend Tornado for agentic use right now. The official .NET package is a good alternative if you intend to only consume OpenAI API. https://github.com/openai/openai-dotnet

**Comment by anwarrassool** [comment]
Thank you. 

The Assistant feature , is that only available with OpenAI ?  or can we create assistants with the other models like Groq and Anthropic

**Comment by lofcz** [comment]
For future readers, as of [v3.2.3](https://github.com/lofcz/LlmTornado/releases/tag/v3.2.3), Agents are now fully supported in LlmTornado.
